### DISCUSSION TITLE
SWR in node environment

### DISCUSSION THREAD

**Discussion by jonyw4** [nuxt/nuxt#16636] [Questions]
Hi there, I am struggling at the moment to understand how I can achieve SWR in node environment. It seems is possible via nitro but I was not able to make it work. Can someone help me?

**Comment by jgupta** [comment]
https://github.com/nuxt/framework/pull/8067 rc12 should bring these changes. It might not work on node environment though (should work on netlify as per comment there), I am not sure.

At present, we are using [Nginx to cache](https://docs.nginx.com/nginx/admin-guide/content-cache/content-caching/) and it is kind of SWR which works nicely for us.

**Reply by jonyw4** [reply]
I tried with rc12 but did not work for me too. 

I was expecting something similar to https://github.com/pi0/nuxt-swr-cache @pi0  implementation but I was not able to find a similar hook or function to trigger the rendering/build in the background

**Reply by jonyw4** [reply]
I was even thinking about creating a nuxt module for swr caching by using an agnostic approach, but I am struggling with the available hooks and the complexity between nuxt app and nitro lifecycle, we don't have too many docs about that yet...
